cli/src/program.rs

```diff
@@ -266,6 +270,24 @@ impl ProgramSubCommands for App<'_, '_> {
                                 .help("Public key of the account to query")
                         ),
                 )
+                .subcommand(
+                    SubCommand::with_name("get-info")
```

Comment:
There was a big push a while ago to strip out all the `get-`s from cli subcommands. I know it has already snuck back in here with "get-authority", but it would be nice to avoid introducing more. Can this just be "info"?
I think "get-authority" can just be "authority" as well (although we can leave "get-authority" as an alias, if you think anyone is using it already.

Reply:
Yup, I'm replacing `get-info` with `show` and removing `get-authority`

Reply:
\o/ awesome!